js/models/stock.js

```diff
@@ -1,106 +1,36 @@
-        "ChangePercent": 0.293925538863489,
-        "Timestamp": "Mon May 4 15:59:00 UTC-04:00 2015",
-        "MSDate": 42128.6659722222,
-        "MarketCap": 188341543260,
-        "Volume": 74680,
-        "ChangeYTD": 530.66,
-        "ChangePercentYTD": 4.16839407530246,
-        "High": 556.61,
-        "Low": 547.6,
-        "Open": 550.1
-    },
-    {
-        "Status": "SUCCESS",
-        "Name": "Microsoft Corp",
-        "Symbol": "MSFT",
-        "LastPrice": 48.24,
-        "Change": 0,
-        "ChangePercent": 0,
-        "Timestamp": "Mon May 4 15:59:00 UTC-04:00 2015",
-        "MSDate": 42128.6659722222,
-        "MarketCap": 390241098000,
-        "Volume": 2875552,
-        "ChangeYTD": 46.45,
-        "ChangePercentYTD": 3.85360602798708,
-        "High": 48.86,
-        "Low": 48.18,
-        "Open": 48.41
-    },
-    {
-        "Status": "SUCCESS",
-        "Name": "3M Co",
-        "Symbol": "MMM",
-        "LastPrice": 159.05,
-        "Change": 0.0300000000000011,
-        "ChangePercent": 0.0188655515029563,
-        "Timestamp": "Mon May 4 15:59:00 UTC-04:00 2015",
-        "MSDate": 42128.6659722222,
-        "MarketCap": 100891936050,
-        "Volume": 153090,
-        "ChangeYTD": 164.32,
-        "ChangePercentYTD": -3.20715676728334,
-        "High": 159.85,
-        "Low": 158.12,
-        "Open": 158.12
-    },
-    {
-        "Status": "SUCCESS",
-        "Name": "Twitter Inc",
-        "Symbol": "TWTR",
-        "LastPrice": 37.89,
-        "Change": 0.00999999999999801,
-        "ChangePercent": 0.0263991552270275,
-        "Timestamp": "Mon May 4 15:59:00 UTC-04:00 2015",
-        "MSDate": 42128.6659722222,
-        "MarketCap": 24794496090,
-        "Volume": 1394555,
-        "ChangeYTD": 35.87,
-        "ChangePercentYTD": 5.63144689155284,
-        "High": 38.96,
-        "Low": 37.56,
-        "Open": 38.7
-    },
+// define your Stock model here
+function Stock(symbol, callback) {
+  this.symbol = symbol;
+  this.numShares = 0;
+
+
+    var markItOnDemand = "http://dev.markitondemand.com/api/v2/quote/jsonp?symbol" + symbol
+    $.ajax
+    ({
+
+      url: markItOnDemand,
+      type: "GET",
+      dataType: "jsonp",
+      context: this
+    }).done(function(response)
     {
-        "Status": "SUCCESS",
-        "Name": "Facebook, Inc.",
-        "Symbol": "FB",
-        "LastPrice": 78.81,
-        "Change": -0.179999999999993,
-        "ChangePercent": -0.227876946448908,
-        "Timestamp": "Mon May 4 15:59:00 UTC-04:00 2015",
-        "MSDate": 42128.6659722222,
-        "MarketCap": 221317079160,
-        "Volume": 839079,
-        "ChangeYTD": 78.02,
-        "ChangePercentYTD": 1.01256088182518,
-        "High": 79.69,
-        "Low": 78.64,
-        "Open": 79.28
-    }
-];
+      for( var property in response )
+      // {
+  //      if( response.prop(property))
+        {
+          this[property] = response [property]
+        }
+      //  StockView.render()
```

Comment:
oh wow, ok, so you want allll the things, that's cool! I like this. I also like the `context: this`, i didn't know you could even do that! The $.ajax doesn't need to be indented also the the variable instantiation on line 7.